app/src/main/java/com/example/ohthmhyh/EditProfile.java

```diff
@@ -0,0 +1,72 @@
+package com.example.ohthmhyh;
```

Comment:
I'd recommend refactoring the name of this file to `EditProfileActivity.java`.

Reply:
resolved